### DISCUSSION TITLE
Is there any way to use multiple API keys?

### DISCUSSION THREAD

**Discussion by danielrosehill** [danny-avila/LibreChat#4831] [Feature Requests & Suggestions]
Hey!

I have two sets of API keys for OpenAI (one for my personal projects and the other for work related stuff).

For Assistants in particular, it would be handy to be able to pass both keys as dot env variables so that I could have all my assistants accessible from the one interface.

Is there any way to do that by any chance?

**Comment by aswny** [comment]
@danny-avila need your help with this one

**Comment by nlelovic** [comment]
Hello, is there an update on this?

**Comment by christopheSeeka** [comment]
I would needs that too, is it possible to have multiple assistants with different asistant api key right now? or not at the moment? Any possible workaround to achieve this?
